### PR TITLE
Lk pd 2363 join barcodes

### PR DESCRIPTION
**Pull request by ekiernan**
### Description
This PR joins ATAC and Optimus barcodes in the respective workflow h5ad files. It also renames the ATAC h5ad index to atac_barcodes. 

----

### Checklist 
If you can answer "yes" to the following items, please add a checkmark next to the appropriate checklist item(s) **and** notify our WARP documentation team by tagging either @ekiernan or @kayleemathews in a comment on this PR.

- [ ] Did you add inputs, outputs, or tasks to a workflow?
- [ ] Did you modify, delete or move: file paths, file names, input names, output names, or task names?
- [ ] If you made a changelog update, did you update the pipeline version number?

### CODE REVIEW

File: pipelines/skylab/optimus/Optimus.wdl [hunk 1]
```diff
@@ -64,7 +64,7 @@ workflow Optimus {
 
   # version of this pipeline
 
-  String pipeline_version = "6.1.0"
+  String pipeline_version = "6.2.0"
```
Comment (nikellepetrillo) [inline pipelines/skylab/optimus/Optimus.wdl:67]: should this be a minor patch if it doesnt actually change the optimus outputs?

**Review by nikellepetrillo — APPROVED** [review]
one small questions, but i think this looks good!

File: pipelines/skylab/multiome/Multiome.wdl [hunk 2]
```diff
@@ -100,6 +107,6 @@ workflow Multiome {
         File cell_metrics_gex = Optimus.cell_metrics
         File gene_metrics_gex = Optimus.gene_metrics
         File? cell_calls_gex = Optimus.cell_calls
-        File h5ad_output_file_gex = Optimus.h5ad_output_file
+        File h5ad_output_file_gex = JoinBarcodes.gex_h5ad_file
```
Comment (kayleemathews) [inline pipelines/skylab/multiome/Multiome.wdl:110]: Does this change the naming of the output file?

**Review by kayleemathews — COMMENTED** [review]
I made a few minor updates to affected docs and I have a question about the naming of a couple outputs but looks great. Thanks for updating the docs!

**Review by kevinpalis — APPROVED** [review]
Looks good to me!

File: pipelines/skylab/multiome/Multiome.wdl (hunk 2 above)
Comment (ekiernan) [reply pipelines/skylab/multiome/Multiome.wdl:110]: No, the actual file names stay the same

File: pipelines/skylab/multiome/Multiome.wdl [hunk 3]
```diff
@@ -89,7 +96,7 @@ workflow Multiome {
         # atac outputs
         File bam_aligned_output_atac = Atac.bam_aligned_output
         File fragment_file_atac = Atac.fragment_file
-        File snap_metrics_atac = Atac.snap_metrics
+        File snap_metrics_atac = JoinBarcodes.atac_h5ad_file
```
[the remark this answers is not in the document]
Comment (ekiernan) [reply pipelines/skylab/multiome/Multiome.wdl:99]: No, the file name stays the same

File: pipelines/skylab/optimus/Optimus.wdl (hunk 1 above)
Comment (ekiernan) [reply pipelines/skylab/optimus/Optimus.wdl:67]: I'll change to minor patch

**Comment by ekiernan** [comment]
> I made a few minor updates to affected docs and I have a question about the naming of a couple outputs but looks great. Thanks for updating the docs!

Thank you for taking a look!

File: website/docs/Pipelines/Optimus_Pipeline/README.md [hunk 4]
```diff
@@ -7,7 +7,7 @@ slug: /Pipelines/Optimus_Pipeline/README
 
 | Pipeline Version | Date Updated | Documentation Author | Questions or Feedback |
 | :----: | :---: | :----: | :--------------: |
-| [optimus_v6.1.0](https://github.com/broadinstitute/warp/releases?q=optimus&expanded=true) | July, 2023 | Elizabeth Kiernan | Please file GitHub issues in warp or contact [the WARP team](mailto:warp-pipelines-help@broadinstitute.org) |
+| [optimus_v6.2.0](https://github.com/broadinstitute/warp/releases?q=optimus&expanded=true) | October, 2023 | Elizabeth Kiernan | Please file GitHub issues in warp or contact [the WARP team](mailto:warp-pipelines-help@broadinstitute.org) |
```
Comment (kayleemathews) [inline website/docs/Pipelines/Optimus_Pipeline/README.md:10]:
```suggestion
| [optimus_v6.1.1](https://github.com/broadinstitute/warp/releases?q=optimus&expanded=true) | October, 2023 | Elizabeth Kiernan | Please file GitHub issues in warp or contact [the WARP team](mailto:warp-pipelines-help@broadinstitute.org) |
```

**Pull request merged**